Makefile

```diff
@@ -1,34 +1,48 @@
 EXTRA_ARGS :=
 
+ifneq ($(OS),Windows_NT)
 UNAME := $(shell uname)
 ifeq ($(UNAME), Linux)
 NPROC = $(shell nproc)
 endif
 ifeq ($(UNAME), Darwin)
 NPROC = $(shell sysctl -n hw.physicalcpu)
 endif
+endif
 
 #########
 # BUILD #
 #########
 .PHONY: requirements develop build build-debug build-conda install
 
 requirements:  ## install python dev and runtime dependencies
+ifeq ($(OS),Windows_NT)
+	Powershell.exe -executionpolicy bypass -noprofile ./make_requirements.ps1
```

Comment:
I'm going to see if we can avoid random top level files/scripts by inlining this

Reply:
I would just move the scripts into a subdirectory and call it a day

Reply:
there is already a `ci` subdirectory, could add `ci/scripts` but i like having everything nice and clean in the makefile 😢

Reply:
I agree but it wasnt trivial, I def gave it a good try before giving up.  Dont want to invest a day figuring it out at this point, but happy to open an issue to have someone else take a crack at it at some point

Reply:
moved to ci\scripts\windows\
opened: https://github.com/Point72/csp/issues/242